decidim-forms/app/packs/src/decidim/forms/admin/forms.js

```
@@ -390,7 +389,8 @@ export default function createEditableForm() {
       autoLabelByPosition.run();
       autoButtonsByPosition.run();
 
-      initLanguageChangeSelect($field.find("select.language-change").toArray());
+      document.querySelectorAll("select.language-change").forEach((container) =>
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)